packages/yoastseo/src/languageProcessing/languages/fr/helpers/checkIfWordIsComplex.js

```diff
@@ -45,16 +46,19 @@ export default function checkIfWordIsComplex( config, word ) {
 	if ( word[ 0 ].toLowerCase() === word[ 0 ] ) {
 		/*
 		 * If a word is longer than 9 characters and doesn't start with capital letter,
-		 * we check further whether it is a plural ending in -s. If it is, we remove the -s suffix
+		 * we check further whether it is a plural that ends on the -s or -x plural suffix. If it is, we remove the plural suffix
 		 * and check if the singular word can be found in the frequency list.
+		 * if it is a plural that does not end on -s or -x but on -aux, we replace the plural -aux suffix with the singular suffix -al.
 		 * The word is not complex if the singular form is in the list.
 		 */
-		if ( word.endsWith( "s" ) ) {
+		if ( word.test( morphologyData.suffixGroupsComplexity.standardSuffixesWithSplural ||
+			morphologyData.suffixGroupsComplexity.standardSuffixesWithXplural ) ) {
```

Comment:
Looking at the `standardSuffixesWithSplural` and `standardSuffixesWithXplural`, they are string. So here, if `.test()` is used, then both variable above also need to be made regex. So something like: `new RegExp( morphologyData.suffixGroupsComplexity.standardSuffixesWithSplural )` and `new RegExp( morphologyData.suffixGroupsComplexity.standardSuffixesWithXplural )`

Additionally, `.test()` is a regex method. This means that this method needs to be attached to the regex itself instead of the string we want to test. See [this article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) for more info.

So the check above can be rewritten as follows:
```
		if ( new RegExp( morphologyData.suffixGroupsComplexity.standardSuffixesWithSplural ).test( word ) ||
			new RegExp( morphologyData.suffixGroupsComplexity.standardSuffixesWithXplural ).test( word ) ) {
```
You can also make it a bit prettier by for example making a separate variable for the regexes, and put them outside of the function (for performance reason)